### PR TITLE
Create FS-1100-Printf-binary.md

### PR DESCRIPTION
**Pull request by Happypig375**

### CODE REVIEW

File: RFCs/FS-1100-Printf-binary.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+its limitations (no signed numbers) to be consistent and be within expectations.
+
+```fs
+printf "%o" 123
+printf "%B" 123
+```
+```
+173
+1111011
+```
```
Comment (cartermp) [inline RFCs/FS-1100-Printf-binary.md:45]:
```suggestion
The following F# code:

```fs
printf "%o" 123
printf "%B" 123
```

Prints:

```
173
1111011
```
```

File: RFCs/FS-1100-Printf-binary.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+# Drawbacks
+
+This will be the first additional printf format specifier introduced to F# since 1.0, so
```
Comment (cartermp) [inline RFCs/FS-1100-Printf-binary.md:49]: Not that big of a drawback :)

**Review by cartermp — APPROVED** [review]
Thanks! Just a quick formatting change.

File: RFCs/FS-1100-Printf-binary.md (hunk 1 above)
Comment (Happypig375) [reply RFCs/FS-1100-Printf-binary.md]: Wait... This messes up the formatting.

**Comment by cartermp** [comment]
Thanks a bunch. Would you also be interested in submitting a PR as well? A place to start would be here: https://github.com/dotnet/fsharp/blob/main/src/fsharp/FSharp.Core/printf.fs#L985

**Comment by Happypig375** [comment]
I'll try, yes

**Pull request merged**